src/lib/Decorator/LocationDecorator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\HybridPlatformUi\Decorator;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+class LocationDecorator extends ValueObjectDecorator
```

Comment:
It's more because the likes of twig do some level of introspection of the code, these 3 methods are defined in the location value object.

the idea here is to implement all public methods and pass them to the original object